Configurations.md

```diff
@@ -3112,12 +3112,10 @@ Note that no wrapping will happen if:
 // Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
 
 // Information on the lorem ipsum can be found at the following url: https://en.wikipedia.org/wiki/Lorem_ipsum. Its text is: lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
-/// # This doc comment is a very long header (it starts with a '#'). Had it not been a header it would have been wrapped. But because it is a header, it will not be. That is because wrapping a markdown header breaks it.
```

Comment:
I'm fine with removing this from the example code, but I'd like to preserve this information by adding it to the note above. I'm thinking something like this:

````markdown
Break comments to fit on the line

Note that no wrapping will happen if:
1. The comment is the start of a markdown header doc comment. For example:
   ```rust
   /// # This doc comment is a markdown header
   ```
3. An URL was found in the comment
````